meson_init.sh

```diff
@@ -105,6 +106,7 @@ for platform in ${PLATFORMS[@]}; do
   mkdir -p "$obj_dir"
   meson ${FLAGS_reconfigure} \
     -Dtarget="$platform" \
+    -Dot_version="$OT_VERSION" \
```

Comment:
What is the developer flow. Does `build_consts.sh` get called at some point before meson_init.sh? It is nice to be able to get a commit stamp when debugging developer issues.

Reply:
Yeah. build_consts.sh gets called at the start of meson_init.sh, and $OT_VERSION is then printed out (see the top of the file).

Reply:
ack, thx